styx-scheduler-service/src/test/java/com/spotify/styx/SystemTest.java

```diff
@@ -594,49 +594,6 @@ public void doNotDeleteWorkflowFromNoneAPI() throws Exception {
   }
 
   @Test
-  public void cleansUpDockerRunsWhenTerminating() throws Exception {
```

Comment:
Should this test be updated to verify that cleanup happens after some grace period using `timePasses()` instead?

Reply:
Ok, seems these tests are now invalid, nvm.